io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -49,6 +49,9 @@
     <content_attribute id="social-contacts">intense</content_attribute>
   </content_rating>
   <releases>
+    <release version="0.21.0 Beta" date="2024-05-20">
```

Comment:
This is the wrong month!

Reply:
Thanks for catching this. Had to update this manually.